src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -43,13 +43,14 @@ internal partial class CurlHandler : HttpMessageHandler
 
         private volatile bool _anyOperationStarted;
         private volatile bool _disposed;
-        private bool _automaticRedirection = true;
         private IWebProxy _proxy = null;
         private ICredentials _serverCredentials = null;
         private ProxyUsePolicy _proxyPolicy = ProxyUsePolicy.UseDefaultProxy;
         private DecompressionMethods _automaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate;
         private SafeCurlMultiHandle _multiHandle;
         private GCHandle _multiHandlePtr = new GCHandle();
+        private bool _automaticRedirection = true;
+        private int _maxAutomaticRedirections = 50;
```

Comment:
There are a bunch of these default values that are being copied from the WinHttp handler implementation.  It's great that we're using the same defaults, but it's error prone that we're doing so via copy-and-paste.  I see a whole bunch of constants/defaults in WinHttp at https://github.com/dotnet/corefx/blob/master/src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpHandler.cs#L93-L146... could we somehow factor out these constants/default values into a shared source file that's included in both the Windows and Unix implementation, such that this then becomes:
```C#
private int _maxAutomaticRedirections = HttpHandlerDefaults.MaxAutomaticRedirections;
```
instead of:
```C#
private int _maxAutomaticRedirections = 50;
```
?
@davidsh, that ok?